docs/tutorials/basics/basics_outlook.md

```diff
@@ -0,0 +1,50 @@
+# Basics of Syne Tune: Outlook
+
+
+[[Previous Section]](basics_backend.md)
+
+
+## Further Topics
+
+We are at the end of this basic tutorial. There are many further topics we did
+not touch here. Some are established, but not basic, while others are still
+experimental. Here is an incomplete overview:
+
+* **Running many experiments in parallel**: We have stressed the importance of
+  running repetitions of experiments, as results carry quite some stochastic
+  variation. Also, there are higher-level decisions best done by trial-and-error,
+  which can be seen as "outer loop random search". Syne Tune offers facilities to
+  launch many tuning experiments in parallel, as SageMaker training jobs. See
+  [README.md](../../../README.md#launching-a-tuning-job) and
+  [launch_height_sagemaker_remotely.py](../../../examples/launch_height_sagemaker_remotely.py).
```

Comment:
Can you add a link to the benchmark loop example as well? Something like
```suggestion
  launch many tuning experiments in parallel, as SageMaker training jobs. See
  [launch_height_sagemaker_remotely.py](../../../examples/launch_height_sagemaker_remotely.py) for an example to launch a tuning job remotely or 
  [launch_height_sagemaker_remotely.py](../../../benchmarks/benchmark_loop/README.md) for an example
  to launch a loop of different combinations remotely.
```

Reply:
Good point. I am also planning to move the CL launcher to use that second option.